docs/smtchecker.rst

```diff
@@ -404,16 +404,60 @@ that the assertion fails:
      | 		^^^^^^^^^^^^^^^^^
 
 
+******************
+SMTChecker Options
+******************
+
+Model Checking Engines
+======================
+
+The SMTChecker module implements two different reasoning engines, a Bounded
+Model Checker (BMC) and a system of Constrained Horn Clauses (CHC).  Both
+engines are currently under development, and have different characteristics.
+The engines are independent and every property warning states from which engine
+it came from. Notice that all the examples above with counterexamples were
```

Comment:
Loose the "from" at the end. Or "and every property warning states which engine issued it."
Also "Note that" instead of "Notice that" maybe?